framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -230,6 +270,27 @@
 	    }
 	    return list;
 	  }
+
+  private static List<File> searchFolders(File root, String regex) {
+          List<File> list = new ArrayList<File>();
+          Pattern pattern = Pattern.compile(regex + "$");
+          Matcher matcher = null;
+          if (!root.isFile()) {
```

Comment:
sorry, shouldn't you prepend the pathPrefix to pathList and then check if it exists or not?